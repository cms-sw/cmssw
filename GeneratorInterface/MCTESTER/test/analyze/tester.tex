\documentclass{article}
\usepackage[T1]{fontenc}
\usepackage[latin1]{inputenc}
\usepackage{graphics}
\usepackage{longtable}
\usepackage{color}
\usepackage{pslatex}
\usepackage{hyperref}
\begin{document}

\include{mc-results}
%\tableofcontents{}
\include{booklet}
\end{document}
