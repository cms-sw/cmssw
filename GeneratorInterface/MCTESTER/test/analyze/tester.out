\BOOKMARK [1][-]{section.1}{Decay Channel: + + 0 0 "0365 }{}
\BOOKMARK [1][-]{section.2}{Decay Channel: + - + + "0365 }{}
